src/utils/formatting.js

```diff
@@ -6,3 +6,7 @@ export const joinNames = (firstName, lastName) => {
 };
 
 export const lowerFirst = key => `${key[0].toLowerCase()}${key.slice(1)}`;
+
+const MINSK_OFFSET = 180 * 60 * 1000;
```

Comment:
i guess that will be broken once Belarus starts switching between summer and winter time again